File: php/tts_token_demo.php
<?php
/**
 * 语音合成服务的token方式请求的示例
 */
require "./ais_sdk/gettoken.php";
require "./ais_sdk/tts.php";
require "./ais_sdk/utils.php";

// region目前支持华北-北京一(cn-north-1)
init_region($region = 'cn-north-1');

$username = "********";      // 配置用户名
$password = "********";      // 密码
$domainName = "*********";   // 配置用户名

$token = get_token($username, $password, $domainName);

// token默认方式请求数据
$result = tts($token, "语音合成为你的业务增加交互的能力.");
$resultobj = json_decode($result, true);
$basestr = $resultobj["result"]["data"];
base64_to_file("data/tts_use_default_config.wav", $basestr);

// token特殊方式请求数据
$result = tts($token, "语音合成为你的业务增加交互的能力.", "xiaoyan", 0, "16k", 0, 0);
$resultobj = json_decode($result, true);
$basestr = $resultobj["result"]["data"];
base64_to_file("data/tts_use_specific_config.wav", $basestr);

File: python3/tts_token_demo.py
# -*- coding:utf-8 -*-

from ais_sdk.gettoken import get_token
from ais_sdk.utils import decode_to_wave_file,init_global_env
from ais_sdk.tts import tts

import json

if __name__ == '__main__':
    # Services currently support North China-Beijing 1 (cn-north-1)
    init_global_env('cn-north-1')

    #
    # access ocr vat invoice,post data by token
    #
    user_name = '*****'
    password = '******'
    account_name = '*****'  # the same as user_name in commonly use

    token = get_token(user_name, password, account_name)

    # call interface use the default config
    result = tts(token, '语音合成为你的业务增加交互的能力.')
    result_obj = json.loads(result)
    decode_to_wave_file(result_obj['result']['data'], 'data/tts_use_default_config.wav')

    # call interface use the specific config
    result = tts(token, '这里是语音合成的测试。', 'xiaoyu', '0', '16k')
    result_obj = json.loads(result)
    decode_to_wave_file(result_obj['result']['data'], 'data/tts_use_specific_config.wav')